backend/src/main/java/com/empire/World.java

```diff
@@ -713,12 +708,13 @@ public double sum() {
 			Region region = regions.get(c.location);
 			if (!"".equals(c.captor)) continue; // inspire can't be done by captives.
 			if (action.equals("Inspire the Faithful") && "Sancta Civitate".equals(region.name)) {
+				c.addExperience("spy", this);
```

Comment:
Trying to keep my editorial comments about the game changes themselves out of this review but...huge ouch to the northerners.

Reply:
Yeah, the rules stated they got Govern XP out of Inspire (which apparently I never implemented), but the southern players pointed out that spy experience matters more, especially since Church intrigue always happens in Sancta Civitate.

It definitely makes the Cardinals more important and tricky to assail once they are established.